master/internal/webhooks/postgres_webhook.go

```diff
@@ -19,8 +21,119 @@ import (
 	"github.com/google/uuid"
 )
 
-// AddWebhook adds a Webhook and its Triggers to the DB.
-func AddWebhook(ctx context.Context, w *Webhook) error {
+type cacheItem struct {
+	re                 *regexp.Regexp
+	triggerIDToTrigger map[TriggerID]*Trigger
+}
+
+type webhookManager struct {
+	mu    sync.RWMutex
+	items map[string]cacheItem
+}
```

Comment:
i'd like to argue away the existence of the cache and import re in webhooks if i can, on a call

Reply:
talked offline and decided current way is fine